src/seedu/addressbook/data/person/Address.java

```diff
@@ -37,22 +58,69 @@ public static boolean isValidAddress(String test) {
 
     @Override
     public String toString() {
-        return value;
+        return blockData.getBlock() + " " + streetData.getStreet() + " " + unitData.getUnit() + " " + postalCodeData.getPostalCode();
```

Comment:
Our coding standard has a limit on how long a line can be. Reason: to avoid having to scroll horizontally.